Lib/unittest/case.py

```diff
@@ -338,6 +339,19 @@ def __exit__(self, exc_type, exc_value, tb):
                 .format(logging.getLevelName(self.level), self.logger.name))
 
 
+def _short_description_from_docstring(text):
+    # Return the summary line from a docstring.
+    # If there is no summary line, return None.
+    if text:
```

Comment:
It would be cleaner to use a guard clause and just write (and with no comment):

    if not text:
        return None